Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.4
+
+ENV GOPATH /go
+ENV PATH ${PATH}:/go/bin
+ENV TARGET /source
+ENV GOPATH_TARGET /go/src/github.com/docker/docker-network
+
+ENV SUBSYSTEMS namespace
+
+RUN go get -u github.com/kr/godep
+RUN mkdir ${TARGET}
+RUN mkdir -p $(dirname ${GOPATH_TARGET}) && \ 
```

Comment:
I think it's probably not cool to assume that source mounted to `/source`. Then it won't be buildable by just docker build `https://github.com/docker/docker-network`. Ideas is to `COPY` source or just `go get` inside Dockerfile.